website/docs/docs/dbt-cloud/on-premises/setup.md

```diff
@@ -84,4 +84,16 @@ Enter the following:
 - Install URL: right click "Public page" on the "About" page, click "Copy Link Location," and paste in the value here.
 - Private Key PEM: on the "About" page of the newly created Github app, scroll down to the bottom of the page. Under "Private Keys," click "Generate a Private Key." Download the key, then open up the file and paste the contents into this field.
 
-After you save your dbt Cloud configuration, you are ready to manage your Github integration. Your users can log into dbt Cloud, and navigate to Profile > Integrations to start connecting your Github account to dbt Cloud. See https://docs.getdbt.com/docs/cloud-installing-the-github-application for more details on how your users can start using the integration.
+3. Save your dbt Cloud configuration.
+4. Restart the "app" pod to update the configuration values.
+##### To do this on kubernetes 1.15+:
+```
+kubectl rollout restart deployment app
+```
+##### To do this on kubernetes <1.15:
+```
+kubectl patch deployment app -p \
+  "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"date\":\"`date +'%s'`\"}}}}}"
+```
```

Comment:
this is strictly an improvement so I'm going to approve this. But, in the long run, it feels like this should be part of a "maintenance" section in the on-premises guide.

Reply:
I agree. I also do think that Github setup (and maybe a guide to setting up a Slack app) should probably live in its own page. It feels a bit random on this one.

Reply:
I think what needs to be done is more information on useful things to know about maintenance.